docs(review): review docs 17/2023 (#1581)
scaleway/docs-content: containers/kubernetes/api-cli/cluster-monitoring.mdx

    [..]
    ```

> 4. Verify that the created pods are all running once the stack is deployed. It is also possible to check if the 100Gi block volume was created:

Reviewer: ```suggestion
4. Verify that the created pods are all running once the stack is deployed. You can also check whether the 100Gi block volume was created:
```

* `persistence.size` : 10Gi

> 1. Add the Helm repository and install Grafana using `helm` with the following commands:

Reviewer: ```suggestion
1. Run the following command to add the Helm repository and install Grafana using `helm`:
```
